<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WifiTestWidgetClass</class>
 <widget class="QWidget" name="WifiTestWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WifiTestWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>WIFI测试</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="WIFIName_LineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="PassWD_LineEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="WifiConnect_Button">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="ssid_label">
         <property name="text">
          <string>ssid：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ssid_lineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="bssid_label">
         <property name="text">
          <string>bssid：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="bssid_lineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="flag_label">
         <property name="text">
          <string>flag：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="flag_lineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="frequency_label">
         <property name="text">
          <string>frequency：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="frequency_lineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="dBm_label">
         <property name="text">
          <string>dBm：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="dBm_lineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="key_mgmt_label">
         <property name="text">
          <string>key_mgmt：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="key_mgmt_lineEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="linkspeed_label">
         <property name="text">
          <string>linkspeed：</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="linkspeed_lineEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="wpa_state_label">
         <property name="text">
          <string>wpa_state：</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="wpa_state_lineEdit"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="ip_label">
         <property name="text">
          <string>ip：</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="ip_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="averageNum_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="min_label">
            <property name="text">
             <string>min:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="minNum_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="max_label">
            <property name="text">
             <string>max:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="maxNum_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="down_Button">
            <property name="text">
             <string>下载</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="up_Button">
            <property name="text">
             <string>上传</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="true_Button">
       <property name="text">
        <string>成功</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="false_Button">
       <property name="text">
        <string>失败</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_info">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
